src/app/basket/basket.component.html

```diff
@@ -13,13 +13,13 @@ <h1>Personal complex collection</h1>
       <tbody>
       <tr *ngFor="let key of getKeys(complexBasket)">
```

Comment:
Should we get rid of the list? Or add list/navigator buttons, similar to the ones in the search?